edxval/serializers.py

```diff
@@ -122,6 +124,21 @@ class Meta:  # pylint: disable=C1001, C0111
         lookup_field = "edx_video_id"
         exclude = ('id',)
 
+    def get_course_video_image_url(self, video):
```

Comment:
Rather than doing this in `VideoSerializer`, we should create `CourseVideoSerializer` (and use it in api), since we intend to use course-specific video thumbnail url as `VideoImage` is having in one to one relation with `CourseVideo`.